### DISCUSSION TITLE
Is there a way to have Organic Maps stay fully offline?

### DISCUSSION THREAD

**Discussion by Decoherent** [organicmaps/organicmaps#8329] [Q&A]
If I'm someplace where mobile internet is really expensive, or something, I'd like to set a map app to only use downloaded, offline maps. I don't see a switch for that anywhere in the app, nor does search really help me. If there's something I've missed, please let me know :)

**Comment by hb0nd** [comment]
> If I'm someplace where mobile internet is really expensive, or something, I'd like to set a map app to only use downloaded, offline maps. I don't see a switch for that anywhere in the app, nor does search really help me. If there's something I've missed, please let me know :)

it is fully offline :)
you can set "Mobile Internet" to "Never Use" though, to not upload edits on mobile data

**Comment by Decoherent** [comment]
Oh, cool, thanks :) Somehow I was expecting a switch or something, coming from the Osmand world! Much appreciated!
